### ISSUE TITLE
Feature request: Save for later/bookmark

### ISSUE THREAD

**Issue by siddharthkp** [SaraVieira/awesome-talks#72]
Sometimes you see a talk you want to watch, but can't do it right now

Save it for later?

**Comment by sudovijay** [comment]
Yes kinda add to watch list, sounds a good idea to me and easy to implement too

**Comment by pksjce** [comment]
Isnt this the same as "Favorite"-ing feature we have right now?

**Comment by siddharthkp** [comment]
How I imagine it: If you like something you ❤️ it, and then others get recommendations based on that (popular talks)

I would like to save a talk to watch later = bookmark

**Comment by sudovijay** [comment]
Bookmark doesn't sound good, it's a video.  Something like `Watch Later` sounds good to me, Favorite is more or less similar to `Liked Videos`. if we really wanna do this all we gotta do is add a icon below Watched talks, everything else will be like fav.

**Comment by pksjce** [comment]
But we currently don't have the concept of a "User" in the app right? How will you store this info beyond the current session?

**Comment by sudovijay** [comment]
We don't need "User" concept for that, its all localStorage / Cookies (which of course has less limit), just like we doing for fav.
NO, wait. we're not doin' anything lol it even doesn't seem to store favs longer than session. is it due to SSR ?

**Comment by sudovijay** [comment]
localStorage seems to work fine, it just we need to re render few stuff when the app did actually get mounted on client side. otherwise there's no point of having fav or watched talks.

**Comment by siddharthkp** [comment]
> But we currently don't have the concept of a "User" in the app right? How will you store this info beyond the current session?

🤔 I'm not sure, how do we store favorites/already seen right now?

**Comment by SaraVieira** [comment]
> 🤔 I'm not sure, how do we store favorites/already seen right now?

Local storage
